templates/CRM/Price/Form/PriceSet.tpl

```diff
@@ -27,7 +27,7 @@
 
     {foreach from=$priceSet.fields item=element key=field_id}
         {* Skip 'Admin' visibility price fields WHEN this tpl is used in online registration unless user has administer CiviCRM permission. *}
-        {if $element.visibility EQ 'public' || ($element.visibility EQ 'admin' && $adminFld EQ true) || $context eq 'standalone' || $context eq 'advanced' || $context eq 'search' || $context eq 'participant' || $context eq 'dashboard'}
+        {if $element.visibility EQ 'public' || $element.visibility !== 'admin' || $isShowAdmin}
```

Comment:
$isShowAdmin => $isShowAdminVisibilityFields ?

Reply:
argh - I switched to the longer variable name as I went along & obviously didn't loop back around